chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/toastMiddleware.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { NotificationScenarios } from "../../enums/NotificationScenarios";
+import { INotificationPaneProps } from "@microsoft/omnichannel-chat-components/lib/types/components/notificationpane/interfaces/INotificationPaneProps";
+import NotificationPaneStateful from "../../../../notificationpanestateful/NotificationPaneStateful";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const createToastMiddleware = (notificationPaneProps: INotificationPaneProps | undefined, endChat: any) => {
```

Comment:
createCustomNotificationToastMiddlewear or createToastMiddlewearForCustomNotification ???

Reply:
This is named `createToastMiddleware` because it overrides the toast. End goal is for this toast middleware to completely override all toasts (not just custom ones) with our LCW in-house pane, hence I believe we should keep it as `createToastMiddleware`. 

There's no existing toastMiddleware before this PR, so no need to specify that it is for custom notifications.